src/lib/battery/battery_params_deprecated.c

```diff
@@ -142,7 +89,7 @@ PARAM_DEFINE_FLOAT(BAT_EMERGEN_THR, 0.05f);
 PARAM_DEFINE_FLOAT(BAT_V_LOAD_DROP, 0.3f);
 
 /**
- * Explicitly defines the per cell internal resistance
+ * This parameter is deprecated. Please use BAT1_R_INTERNAL instead.
```

Comment:
But `BAT1_R_INTERNAL` is not available yet, right?

Reply:
https://github.com/ItsTimmy/Firmware/blob/612ae0c74341171d4e665dbbaaaacc97b46960d5/src/lib/battery/module.yaml#L61-L77